test/fixtures/es-modules/package-without-pjson/index.js

```diff
@@ -0,0 +1,3 @@
+const identifier = 'package-without-pjson';
```

Comment:
this needs to ensure not package.json exists above it https://github.com/nodejs/node/blob/fc4a7927288048089eaa6bf55177d7b4a3463609/test/common/index.js#L738

Reply:
done with https://github.com/nodejs/node/pull/40980/commits/59350655e1d29fee1c56e6c0e194f28aed49b88d